src/pass/storage_flatten.cc

```diff
@@ -96,11 +96,15 @@ class StorageFlattener : public IRMutator {
       return this->Mutate(op->body);
     } else if (op->attr_key == attr::opengl_stage_scope) {
```

Comment:
can we pass create_bound_attributes as argument of storage flatten?

Reply:
Ok, fixed, but this also required to change public api for Storage Flatten pass and some tests.